src/GameData.ts

```diff
@@ -146,7 +146,8 @@ export default class GameData {
                 result = this.findPerk(searchTerm);
                 break;
             }
-            case "champion": {
+            case "champion": // fall through
+            case "champ": {
```

Comment:
isnt there a fancy ES2018 thing we can do to merge the cases? like `case: ["champ", "champion"]`

In other langs you can do things like this, so i assume JS has something similar?
```

   final String numericString = switch (integer)  
      {  
         case 0 -> "zero";  
         case 1, 3, 5, 7, 9 -> "odd";  
         case 2, 4, 6, 8, 10 -> "even";  
         default -> "N/A";  
      };  
```

Reply:
Nope. Fallthrough is the best way.